azkaban-common/src/test/java/azkaban/ServiceProviderTest.java

```diff
@@ -39,6 +39,15 @@
 
   public static final String AZKABAN_LOCAL_TEST_STORAGE = "AZKABAN_LOCAL_TEST_STORAGE";
 
+  // Test if one class is singletonly guiced. could be called by
+  // AZ Common, Web, or Exec Modules.
+  public static void assertSingleton(final Class azkabanClass, final Injector injector) {
+    assertThat(injector.getInstance(azkabanClass)).isNotNull();
+    final Object azkabanObj1 = injector.getInstance(azkabanClass);
+    final Object azkabanObj2 = injector.getInstance(azkabanClass);
+    assertThat(azkabanObj1).isEqualTo(azkabanObj2);
```

Comment:
Shouldn't it be http://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractAssert.html#isSameAs(java.lang.Object) 
isSameAs?